app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import DisplayValue from '../../../../../components/FormComponents/DisplayValue'
+
+const InformationItem = ({ label, value, showIfMissing }) => {
+  if (!showIfMissing && !value) {
+    return null
+  }
+
+  return (
+    <div className="row border-bottom-line">
```

Comment:
Good opportunity to ditch this grid and use `<GridRow />` and `<GridCol />` from govuk-react.  Will come and help you.

Reply:
moved away from using old grid stuff with John's help